app/routes/public/speakers.js

```diff
@@ -65,7 +65,10 @@ export default class SpeakersRoute extends Route {
         ]
       });
     }
+    const { success, token } = await this.loader.load(`/events/${eventDetails.id}/chat-token`);
```

Comment:
Wrong logic, first this is duplication of code. Secondly, this should be fetched in the component. That's why it is logging in user before opening the chat